project/.bloop/help-to-save-proxy-build.json

```diff
@@ -0,0 +1,2506 @@
+{
```

Comment:
out of curosity , what this file does? it has fiile path specific to your machine